doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -646,7 +653,7 @@ This section provides detailed lists of changes by :ref:`integration <integratio
 Google Fast Pair integration
```

Comment:
My small observation:

We had this "Integrations" section in NCS 2.7 documentation but removed it for NCS 2.8.
It seems that it is back again. 

I think we should consistently use one solution. It may be confusing to the users to sometimes report these updates in the "Integrations" section and other times in the "Documentation" section.

Reply:
@divipillai, what should we do with that?

Reply:
The process we follow is, While create the release notes PR, if there are no entries for a section, we remove that section. This could have been the case. But, the team can surely point to the correct sections the entries belong to (or add new sections) like we normally do for the Changelog file. 
In this case, I agree, we should keep updates in one place and would say the "Integrations" section would be apt.

Reply:
Okay, thanks~I'm leaving it as is then.

Reply:
Maybe, I wasn't specific enough.

In NCS 2.7 we documented the changes to the Google Fast Pair integration guide as part of the Integrations section.

In NCS 2.8 we documented the changes to the same content (the Google Fast Pair integration guide) as part of the Documentation section.